swabbie-aws/src/main/kotlin/com/netflix/spinnaker/config/AwsConfiguration.kt

```diff
@@ -44,6 +47,12 @@ import java.time.Clock
 open class AwsConfiguration {
   private val defaultRegion = "us-west-2" // TODO: (Jeyrs) Make configurable
 
+  // AWS object mapper ensures edda and vanilla aws responses are the same
+  @Bean
+  @Qualifier("amazonObjectMapper")
```

Comment:
You don't need `@Qualifier`, as the name of the factory method is the same value.